tiledb/oxidize/CMakeLists.txt

```
@@ -0,0 +1,164 @@
+# assumes cargo is found via `Modules/FindCargo.cmake`
+
+if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(CARGO_PROFILE "dev")
+    set(CARGO_TARGET "debug")
+else ()
+    set(CARGO_PROFILE "release")
+    set(CARGO_TARGET "release")
+endif ()
```

```suggestion
set(CARGO_PROFILE "$<IF:$<CONFIG:Debug>,dev,release>")
set(CARGO_TARGET "$<IF:$<CONFIG:Debug>,debug,release>")
```

We should not use the `CMAKE_BUILD_TYPE` variable to check for the configuration; it is not compatible with multi-config generators.

---

Done 3da238b